src/nav/apm.yml

```diff
@@ -1103,79 +1103,105 @@ pages:
             pages:
-              - title: Requirements and supported frameworks
-                path: /docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks
+              - title: Latest release
+                path: /docs/release-notes/agent-release-notes/ruby-release-notes/current
+              - title: Ruby release notes
+                path: /docs/release-notes/agent-release-notes/ruby-release-notes
               - title: Ruby agent security
                 path: /docs/apm/agents/ruby-agent/getting-started/apm-agent-security-ruby
               - title: Ruby agent EOL policy
                 path: /docs/apm/agents/ruby-agent/getting-started/ruby-agent-eol-policy
               - title: New Relic's GitHub repository
                 path: /docs/apm/agents/ruby-agent/getting-started/new-relics-github-repository
-              - title: Latest release
-                path: /docs/release-notes/agent-release-notes/ruby-release-notes/current
-              - title: Ruby release notes
-                path: /docs/release-notes/agent-release-notes/ruby-release-notes
               - title: Ruby agent 7x migration guide
                 path: /docs/apm/agents/ruby-agent/getting-started/migration-7x-guide
               - title: Ruby agent 8x migration guide
                 path: /docs/apm/agents/ruby-agent/getting-started/migration-8x-guide
-          - title: Installation
+          - title: Configuration and other install
             path: /docs/apm/agents/ruby-agent/installation
             pages:
-              - title: Ruby agent installation
-                path: /docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent
               - title: Rails plugin installation
                 path: /docs/apm/agents/ruby-agent/installation/ruby-agent-installation-rails-plugin
-              - title: GAE flex
-                path: /docs/apm/agents/ruby-agent/installation/install-new-relic-ruby-agent-gae-flexible-environment
               - title: Ruby agent and Heroku
                 path: /docs/apm/agents/ruby-agent/installation/ruby-agent-heroku
               - title: Update Ruby agent
                 path: /docs/apm/agents/ruby-agent/installation/update-ruby-agent
               - title: Uninstall Ruby agent
                 path: /docs/apm/agents/ruby-agent/installation/uninstall-ruby-agent
-          - title: Configuration
-            path: /docs/apm/agents/ruby-agent/configuration
-            pages:
-              - title: Ruby configuration
-                path: /docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration
               - title: Custom SSL certificates
                 path: /docs/apm/agents/ruby-agent/configuration/custom-ssl-certificates-ruby
               - title: Connect hosts to account
                 path: /docs/apm/agents/ruby-agent/configuration/connect-hosts-your-account
               - title: Distriubted tracing
```

Comment:
@homelessbirds Seems to be a spelling error here. Can we update this to say 'Distributed tracing'

Reply:
Yes, good catch. Thanks!